packages/pojo-router/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 0.2.0 (2020-08-04)
```

Comment:
version bumps and changelogs happen autmatically when runnign 'yarn release' no need to include them here.